.github/workflows/release-embedding-sdk.yml

```diff
@@ -190,27 +189,39 @@ jobs:
           VERSION=$(jq -r .version ./enterprise/frontend/src/embedding-sdk-bundle/package.template.json)
           echo "sdk_current_version=$VERSION" >> $GITHUB_OUTPUT
 
-      - name: Get next SDK patch version
+      - name: Get next SDK patch or pre-release version
         id: new-sdk-version
         uses: actions/github-script@v7
         with:
           script: |
             const currentVersion = "${{ steps.current-sdk-version.outputs.sdk_current_version }}";
             const [currentVersionWithoutSuffix, suffix] = currentVersion.split("-");
             const versionParts = currentVersionWithoutSuffix.split(".");
-            versionParts[versionParts.length - 1] = parseInt(versionParts.at(-1)) + 1;
-            const newVersion = versionParts.join(".");
             const [_, majorVersion] = versionParts
 
-            if (!suffix) {
+            if (branch === "master") {
+              if (!suffix) {
+                throw new Error("Expected pre-release suffix on master branch, got: " + currentVersion);
+              }
+
+              const suffixParts = suffix ? suffix.split(".") : [];
+
+              const preReleaseLabel = suffixParts[0] ?? "";
+              const preReleaseVersion = suffixParts[1] ? parseInt(suffixParts[1]) : 0;
+              const nextPreReleaseVersion = preReleaseVersion + 1;
+
               return {
-                version: newVersion,
-                majorVersion
+                version: `${currentVersionWithoutSuffix}-${preReleaseLabel}.${nextPreReleaseVersion}`,
+                "pre-release-label": preReleaseLabel,
+                majorVersion,
               };
             }
 
+            versionParts[versionParts.length - 1] = parseInt(versionParts.at(-1)) + 1;
```

Comment:
Can we give an example in an inline comment? e.g. `0.57.0-alpha.1 -> 0.57.0-alpha.2`. Just makes it easier to visualize if a reader has never seen the format before.

Reply:
Actually this line increases a patch version
```
// 0.57.0 => 0.57.1
// 0.57.0-alpha -> 0.57.1-alpha
```